<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BilibiliDMB</class>
 <widget class="QMainWindow" name="BilibiliDMB">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Bilibili弹幕板</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="roomidLable">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>72</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>直播间ID：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="roomid">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>20</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>20</number>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTextBrowser" name="giftBoard">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>761</width>
      <height>221</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <bold>true</bold>
      <stylestrategy>PreferAntialias</stylestrategy>
     </font>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::AdjustIgnored</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="connectButton">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>20</y>
      <width>89</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>连接</string>
    </property>
   </widget>
   <widget class="QPushButton" name="disconnectButton">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>20</y>
      <width>89</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>断开</string>
    </property>
   </widget>
   <widget class="QPushButton" name="saveHistoryButton">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>20</y>
      <width>89</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>保存历史</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="dmBoard">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>290</y>
      <width>761</width>
      <height>221</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <bold>true</bold>
      <stylestrategy>PreferAntialias</stylestrategy>
     </font>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
   </widget>
   <widget class="QLabel" name="entryLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>520</y>
      <width>761</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <bold>true</bold>
      <stylestrategy>PreferAntialias</stylestrategy>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="clearButton">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>20</y>
      <width>89</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>清空消息</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuBilibili">
    <property name="title">
     <string>菜单</string>
    </property>
    <addaction name="actionSetting"/>
   </widget>
   <addaction name="menuBilibili"/>
  </widget>
  <action name="actionSetting">
   <property name="text">
    <string>设置</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>roomid</tabstop>
  <tabstop>connectButton</tabstop>
  <tabstop>disconnectButton</tabstop>
  <tabstop>clearButton</tabstop>
  <tabstop>saveHistoryButton</tabstop>
  <tabstop>giftBoard</tabstop>
  <tabstop>dmBoard</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
